Extensions/Content/Dnn.PersonaBar.Extensions/WebApps/Pages.Web/src/components/App.jsx

```diff
@@ -1603,7 +1603,8 @@ class App extends Component {
                                         pageInContextComponents={props.pageInContextComponents}
                                         NoPermissionSelectionPageId={this.noPermissionSelectionPageId}
                                         CallCustomAction={this.CallCustomAction.bind(this)}
-                                        enabled={!((selectedPage && selectedPage.tabId === 0) || inSearch)} />
+                                        enabled={!((selectedPage && selectedPage.tabId === 0) || inSearch)}
+                                        multiplePagesAdded = {this.props.multiplePagesAdded} />
```

Comment:
Don't add spaces before and after equals character; Keep formatting for attributes as all attributes below are doing,